Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/command/DefaultCommandFactoryTest.java

```diff
@@ -131,4 +141,12 @@ public void testCreateResizeCommand() {
 		Command command = commandFactory.createResizeCommand(10, 20);
 		assertThat(command, is(instanceOf(ResizeCommand.class)));
 	}
+
+	@Test
+	public void testCreateColorChangedCommand() {
+		ToolReference toolReference = new DefaultToolReference();
+		MockitoAnnotations.initMocks(this);
+		Command command = commandFactory.createColorChangedCommand(toolReference, bottomNavigationViewHolder, Color.WHITE);
+		assertThat(command, is(instanceOf(ColorChangedCommand.class)));
+	}
```

Comment:
this test seems to fail. 
createColorChangedCommand wants a context as second parameter.
Therefore jenkins can't run the ticket.